oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/registration/ClientRegistration.java

```diff
@@ -46,6 +50,12 @@
 public final class ClientRegistration implements Serializable {
 
 	private static final long serialVersionUID = SpringSecurityCoreVersion.SERIAL_VERSION_UID;
+	private static final Log logger = LogFactory.getLog(ClientRegistration.class);
+
+	private static final List<AuthorizationGrantType> authorizationGrantTypes =
```

Comment:
Can this be adjusted to be a constant name (all caps, e.g. `AUTHORIZATION_GRANT_TYPES`)?

Reply:
I would also suggest moving this down to the `Builder`.

Reply:
(Clarification: I would suggest moving both lines down into the `Builder` and the logger would reference `ClientRegistration.Builder.class` instead.)